dali/kernels/kernel_req.h

```diff
@@ -63,50 +42,11 @@ struct KernelRequirements {
   KernelRequirements &AddInputSet(const KernelRequirements &new_req, bool reuse_scratch,
                                   int alignment = 64) {
```

Comment:
Do we still need this? The doc doesn't say what's the point of this, it silently add the output_shapes, and it's never used.

Reply:
A follow-up, perhaps? It's a standalone (and trivial) change to remove an unused function.

Reply:
@klecki removed